FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SetUp/content.txt

```diff
@@ -1,7 +1,9 @@
 !|dbfit.TeradataTest|
 
+|Connect Using File|!-TestDbConnectionDbFitTeradata.properties-!|
```

Comment:
I'll try that again. I think I was getting an error in FitNesse because of the CamelCase (FitNesse thinks it's a reference to another page).